drivers/SmartThings/matter-lock/src/aqara-lock/init.lua

```diff
@@ -19,11 +19,13 @@ local device_lib = require "st.device"
 local DoorLock = clusters.DoorLock
 local AQARA_MANUFACTURER_ID = 0x115f
 local U200_PRODUCT_ID = 0x2802
+local U300_PRODUCT_ID = 0x2801
 
 local function is_aqara_products(opts, driver, device)
   if device.network_type == device_lib.NETWORK_TYPE_MATTER and
       device.manufacturer_info.vendor_id == AQARA_MANUFACTURER_ID and
-      device.manufacturer_info.product_id == U200_PRODUCT_ID then
+      (device.manufacturer_info.product_id == U200_PRODUCT_ID or
```

Comment:
@HunsupJung if *all Aqara locks* are going to be using this sub driver, we can remove the check for the product ID and just check for the vendor ID. This would prevent us from needing to modify the sub driver in the future whenever we have a new Aqara fingerprint to add.

For example, in the eve-energy sub driver, we only check for the vendor ID because all Eve products will use the sub driver: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/731d2aebdd763d72d9f5951d9456be5b04e16ffc/drivers/SmartThings/matter-switch/src/eve-energy/init.lua#L54

Can you confirm if all Aqara locks will use this sub driver? If yes, then I think we can remove the check for the product_id and just use vendor_id

Reply:
@ctowns 
I agree with your opinion. Ultimately, the goal is to apply it without a sub driver. But now, it would be better to let all Aqara door locks use the sub driver for stability. we can remove the check for the product_id and just check vendor_id.

Reply:
I made this change here: 728b21afc4f90cd0f0e776a82d52f93177802d00

This should simplify further additions of Aqara lock devices so that they can be fingerprint-only changes.